Client/src/js/views/OwnersDetail.jsx

```diff
@@ -524,22 +523,23 @@ var OwnersDetail = React.createClass({
             </Well>
             <Well>
               <h3 className="clearfix">Equipment ({ owner.numberOfEquipment }) <span className="pull-right">
-                <OverlayTrigger trigger="click" placement="top" rootClose overlay={ <Confirm onConfirm={ this.equipmentVerifyAll }></Confirm> }>
-                  <TooltipButton disabled={!isApproved} disabledTooltip={restrictEquipmentVerifyTooltip} className="mr-5" title="Verify All Equipment" bsSize="small">Verify All</TooltipButton>
-                </OverlayTrigger>
+                <CheckboxControl id="showAttachments" className="mr-5" inline updateState={this.updateState}><small>Show Attachments</small></CheckboxControl>
+                <TooltipButton disabled={ !isApproved } disabledTooltip={ restrictEquipmentVerifyTooltip } className="mr-5" title="Verify All Equipment" bsSize="small" onClick={ this.equipmentVerifyAll }>Verify All</TooltipButton>
                 <TooltipButton disabled={ !isApproved } disabledTooltip={ restrictEquipmentAddTooltip } title="Add Equipment" bsSize="small" onClick={ this.openEquipmentDialog }><Glyphicon glyph="plus" /></TooltipButton>
               </span></h3>
               {(() => {
                 if (this.state.loading) { return <div className="spinner-container"><Spinner/></div>; }
 
                 if (!owner.equipmentList || owner.equipmentList.length === 0) { return <Alert bsStyle="success">No equipment</Alert>; }
 
-                var equipmentList = _.orderBy(owner.equipmentList, [this.state.uiEquipment.sortField], [this.state.uiEquipment.sortDesc ? 'desc' : 'asc']);
+                var equipmentList = _.sortBy(owner.equipmentList, this.state.uiEquipment.sortField);
```

Comment:
This line is un-doing a [commit](https://github.com/bcgov/hets/commit/92db2c10ea944e9baee3246848a7c21450576ecf) which uses `_.orderBy` to properly sort the `localArea.name` which was added as part of HETS-1032.